components/Modal/modal.css

```diff
@@ -0,0 +1,158 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Julius+Sans+One&display=swap');
+
+:root{
```

Comment:
Everything looks good, but you can define these variables in your main file itself or rather some custom variable css file and then just import it wherever need be

Reply:
thank you for your suggestion, I will do that from next time